plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Blobstores.js

```diff
@@ -146,12 +151,7 @@ Ext.define('NX.coreui.controller.Blobstores', {
     var me = this,
         win = form.up('nx-coreui-blobstore-add');
 
-    if (win) {
-      me.loadStoreAndSelect(action.result.data.id, true);
-    }
-    else {
-      me.loadStore(Ext.emptyFn);
-    }
+    me.getStore('NX.coreui.store.Blobstore').load();
```

Comment:
I believe you would need to change the stores definition to use just 'Blobstore' in this case, not sure if there's a syntactic variation to support changing dot notation into camel case for this - but it would certainly be ugly ;)